shippable/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM unit9/base:latest
```

Comment:
Need this image for building on Shippable. The base image they provide has old Node, and for pushing builds to repo we need awscli/gcloud.